crates/optimism/hardforks/src/hardfork.rs

```diff
@@ -35,6 +35,8 @@ hardfork!(
         Holocene,
         /// Isthmus: <https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/isthmus/overview.md>
         Isthmus,
+        /// Interop: <https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/interop/overview.md>
```

Comment:
link is 404. better add a code comment `// todo: add link to specs when available `